io.github.trevorsandy.LPub3D.json

```
@@ -0,0 +1,326 @@
+        "if test -f ${FLATPAK_DEST}/build_staging/lpub3dldrawunf.zip; then echo \"Unofficial LDraw library staged to ${FLATPAK_DEST}/build_staging/lpub3dldrawunf.zip.\"; fi",
+        "if test -f ${FLATPAK_DEST}/build_staging/tenteparts.zip; then echo \"TENTE LDraw library staged to ${FLATPAK_DEST}/build_staging/tenteparts.zip.\"; fi",
+        "if test -f ${FLATPAK_DEST}/build_staging/vexiqparts.zip; then echo \"VEX LDraw library staged to ${FLATPAK_DEST}/build_staging/vexiqparts.zip.\"; fi",
+        "if test -d ${FLATPAK_DEST}/build_staging/ldraw; then echo \"LDraw library extracted to ${FLATPAK_DEST}/build_staging/ldraw.\"; fi"
+      ]
+    },
+    {
+      "name": "ldglite",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "QMAKE_LIBDIR=/app/lib",
+        "CONFIG+=release",
+        "CONFIG+=3RD_PARTY_INSTALL=build_staging",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldglite.git",
+          "commit": "de4b06b718dbc67d495f316bd0143f6e59bc3f84"
+        }
+      ],
+      "post-install": [
+        "mv -f /run/build/ldglite/app/build_staging/* ${FLATPAK_DEST}/build_staging/",
+        "if test -d ${FLATPAK_DEST}/build_staging/ldglite-1.3; then echo \"LDGLite staged to ${FLATPAK_DEST}/build_staging/ldglite-1.3.\"; fi"
+      ]
+    },
+    {
+      "name": "ldview",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "QMAKE_LIBDIR=/app/lib",
+        "CONFIG+=release",
+        "CONFIG+=3RD_PARTY_INSTALL=build_staging",
+        "CONFIG+=USE_SYSTEM_LIBS",
+        "CONFIG+=BUILD_CUI_ONLY",
+        "CONFIG+=BUILD_FLATPAK",
+        "CONFIG+=BUILD_TINYXML",
+        "CONFIG+=BUILD_GL2PS",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldview.git",
+          "commit": "fc21e23dd7de24c07f0f37dc7cc36dd5bd252f3c"
+        }
+      ],
+      "post-install": [
+        "mv -f /run/build/ldview/OSMesa/build_staging/* ${FLATPAK_DEST}/build_staging/",
+        "if test -d ${FLATPAK_DEST}/build_staging/ldview-4.4; then echo \"LDView staged to ${FLATPAK_DEST}/build_staging/ldview-4.4.\"; fi"
+      ]
+    },
+    {
+      "name": "povray",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cd unix && chmod +x prebuild3rdparty.sh && ./prebuild3rdparty.sh",
+        "./configure COMPILED_BY=\"Trevor SANDY <trevor.sandy@gmail.com> using FlatHub for LPub3D.\" --prefix=${LP3D_DIST_DIR_PATH} LPUB3D_3RD_PARTY=yes --with-libsdl2 --with-boost-libdir=${FLATPAK_DEST}/lib --enable-watch-cursor",
+        "make",
+        "make check",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/povray.git",
+          "commit": "54b65529090b4161193df24f0d846046b1495f0b"
```

```suggestion
          "commit": "59f04740497a259d94922eeebc4ccff34e7510f2"
```

This revision seems to work.